src/lib/components/workflow/filter-search/list-filter.svelte

```diff
@@ -31,8 +31,8 @@
     placeholder="{translate('type-or-paste-in')} {$filter.attribute}"
   />
   <Button
-    variant="search"
-    unroundLeft
+    variant="secondary"
+    borderRadiusModifier="square-left"
```

Comment:
I think a prop is better so that as much of the css that affects how a component looks is in one place, i.e. the component itself. IMO adding a class to a component should be a last resort and only happen in extreme one-off cases, which we should probably try to avoid.

Reply:
To the core problem, _why_ would we need to manually pass this prop _or_ class?

Reply:
Because some of our interfaces make use of core components, like dropdown menu, button, input, etc. but need to be tweaked slightly for display with other components. I've yet to come up with a better way to solve this apart from making whole new sets of components that have square and/or borderless left or right edges, but it's on my back burner for sure.


https://github.com/temporalio/ui/assets/11775628/b5b55d9d-8f43-4018-9d79-067a3c8014dd


https://github.com/temporalio/ui/assets/11775628/f16b5985-8b9b-47db-8187-f2132e9e50c7